### PR TITLE
LRQA-69361 CanSupportALinkToAURL test fix

### PR DESCRIPTION
**Pull request by leticiachiu**

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase [hunk 1]
```diff
@@ -849,22 +848,10 @@ definition {
...
-		Click(locator1 = "CKEditor#OK_BUTTON");
-
-		Click(locator1 = "Button#SAVE");
-
-		Alert.viewSuccessMessage();
+		FormFields.addParagraphLinkToAURL(paragraphLink = "www.liferay.com");
+		Form.save();
 
-		Click(locator1 = "Icon#BACK");
+		FormsAdminNavigator.gotoBack();
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase:855]: Good replacement, @leticiachiu! :wink:

File: portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro [hunk 2]
```diff
@@ -2,6 +2,40 @@ definition {
...
+		SelectFrame.selectFrameNoLoading(locator1 = "FormFields#PARAGRAPH_LINK_MODAL");
+
+		SelectFrame(value1 = "relative=top");
+
+		Type(
+			fieldLabel = "URL",
+			locator1 = "FormFields#PARAGRAPH_LINK_TEXT_INPUT",
+			value1 = "${paragraphLink}");
+
+		if (isSet(displayText)) {
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro:19]: @leticiachiu, what is the purpose of `displayText`? Is it necessary to fix the test?

File: portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro [hunk 3]
```diff
@@ -2,6 +2,40 @@ definition {
...
+			Type(
+				fieldLabel = "Display Text",
+				locator1 = "FormFields#PARAGRAPH_LINK_TEXT_INPUT",
+				value1 = "${displayText}");
+		}
+
+		if (isSet(linkProtocol)) {
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro:26]: @leticiachiu, what is the purpose of `linkProtocol`? Is it necessary to fix the test?

**Pull request closed without merging**